Clarify Introduction of Wildcard Pattern in F# Pattern Matching Documentation (#44154)
dotnet/docs: docs/fsharp/tour.md

[!code-fsharp[PatternMatching](~/samples/snippets/fsharp/tour.fs#L717-L743)]

> Something you may have noticed is that the `_` pattern is introduced in the following example. This is known as the [Wildcard Pattern](language-reference/pattern-matching.md#wildcard-pattern), which is a way of saying "I don't care what something is". Although convenient, you can accidentally bypass Exhaustive Pattern Matching and no longer benefit from compile-time enforcements if you aren't careful in using `_`. It is best used when you don't care about certain pieces of a decomposed type when pattern matching, or the final clause when you have enumerated all meaningful cases in a pattern matching expression.

Reviewer: ```suggestion
The following sample introduces the `_` pattern, which you might have noticed before. This is known as the [Wildcard Pattern](language-reference/pattern-matching.md#wildcard-pattern), which is a way of saying "I don't care what something is". Although convenient, you can accidentally bypass Exhaustive Pattern Matching and no longer benefit from compile-time enforcements if you aren't careful in using `_`. It is best used when you don't care about certain pieces of a decomposed type when pattern matching, or the final clause when you have enumerated all meaningful cases in a pattern matching expression.
```

Reviewer: Also thinking about just getting rid of the 'noticing' aspect here, does it really add any value?
